guides/release/configuring-ember/debugging.md

```diff
@@ -68,7 +68,7 @@ export default class App extends Application {
 ```
 ### Dealing with deprecations
 
-In addition to what is described in the [Handling Deprecations guide](./handling-deprecations),
+In addition to what is described in the [Handling Deprecations guide](../handling-deprecations),
```

Comment:
```suggestion
In addition to what is described in the [Handling Deprecations guide](../handling-deprecations/),
```

maybe it needs a trailing slash like `[changing the
router's `location`](../specifying-url-type/)` in https://raw.githubusercontent.com/ember-learn/guides-source/master/guides/release/configuring-ember/embedding-applications.md